index.bs

```diff
@@ -6015,7 +6018,11 @@ Changes since earlier versions</h3>
 	"COUNCIL-REPORT-INDEX": {
 		"href": "https://www.w3.org/about/council/",
 		"title": "Council Report Index"
-	}
+	},
+        "AC-FORUM": {
```

Comment:
```suggestion
        "AC-FORUM-POLICY": {
```

Reply:
I'd suggest changing the shorthand of this bibliographical reference, since it isn7t referring to AC Forum itself, but to a policy about it.